tests/aws/services/stepfunctions/v2/test_state/test_test_state_scenarios.py

```diff
@@ -11,7 +11,7 @@
     ServicesTemplates as ST,
 )
 from tests.aws.services.stepfunctions.templates.test_state.test_state_templates import (
-    TestStateTemplate as TCT,
+    TestStateTemplate as TST,
```

Comment:
nice extra catch ✨ 💯 
(I got confused by this as well)

Reply:
Thank you!